pkg/build/e2e-playwright/main.go

```diff
@@ -144,8 +144,8 @@ func run(ctx context.Context, cmd *cli.Command) error {
 			"public/app/core/icons/cached.json",
 
 			// packages we use in playwright tests
-			"packages",           // TODO: do we need all of this?
-			"public/app/plugins", // TODO: do we need all of this?
+			"packages", // TODO: do we need all of this?
+			"public",   // TODO: do we need all of this?
```

Comment:
one of the enterprise tests imports from public/app/extensions/querylibrary. feels like the public dir should be included - e2e tests could easily import util functions from it.

Reply:
maybe we can remove the TODO comment from these at this point?